test/e2e/events_test.go

```diff
@@ -41,7 +45,7 @@ var _ = t.Describe("events", func() {
 		Expect(session.Out).ToNot(Say("unknown method GetContainerEvents")) // no errors
 
 		// Then
-		session.Terminate()
+		defer session.Terminate()
```

Comment:
need to clean up tear down sequence.

1. call defer right after getting the session. SO it will be terminated no matter what
2. defer will execute after stopping crio as written now. Need to move stopping crio somewhere. I think the best can be done is moving StopCrio to AfterEach.

Reply:
This is done. I had to split tests into two suites: 1) one that relies on cri-o 2) a set of options validation tests that don't need to start cri-o. I hope this is fine.